src/AppInstallerCLI.sln

```diff
@@ -606,23 +606,23 @@ Global
 		{846FB88B-BF1B-4F33-9883-E589CEC99739}.TestRelease|ARM64.ActiveCfg = Release|Any CPU
 		{846FB88B-BF1B-4F33-9883-E589CEC99739}.TestRelease|x64.ActiveCfg = Release|Any CPU
 		{846FB88B-BF1B-4F33-9883-E589CEC99739}.TestRelease|x86.ActiveCfg = Release|Any CPU
-		{68808357-902B-406C-8C19-E8E26A69DE8A}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{68808357-902B-406C-8C19-E8E26A69DE8A}.Debug|ARM64.ActiveCfg = Debug|x64

```

Comment:
```suggestion
		{68808357-902B-406C-8C19-E8E26A69DE8A}.Debug|ARM64.ActiveCfg = Debug|ARM64
```
Shouldn't all of these ARM64 configs point to ARM64?

Reply:
The tests project only has x86 and x64 configured, it's not built in arm64. This is why the configuration manager is complaining it can't find debug|arm64 configuration.